src/extensions/text/store/resolvers.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Wordpress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+const resolvers = {
+	getFontUrl:
+		(fontName, fontData) =>
+		async ({ dispatch }) => {
+			const fontUrl = JSON.parse(
```

Comment:
Looks like JSON.parse is unnecessary if avoid double encoding by removing wrapping url in [class-maxi-api](https://github.com/maxi-blocks/maxi-blocks/blob/b66b98a04a5a02e301fb8ece48e692239f372747/core/class-maxi-api.php#L704)

Reply:
sure, this works :+1: